odoo/openupgrade/openupgrade_loading.py

```diff
@@ -157,6 +157,8 @@ def isrelated(model, k):
                     sorted([x[0] for x in v.selection]))
             else:
                 properties['selection_keys'] = 'function'
+        elif v.type == 'binary':
+            properties['attachment'] = str(getattr(v, "attachment", False))
```

Comment:
Seeing the code of `compare.py`, we only need to keep the same set of properties by field on both v10/v11, but it's not needed to have `attachment=False` on the rest of the field types. That's why I put this code and not a `False` value on the default dict.